### PR TITLE
Cheritest errc

### PR DESCRIPTION
**Pull request by bsdjhb**

### CODE REVIEW

File: bin/cheritest/cheritest_util.c [hunk 1]
```diff
@@ -92,12 +92,23 @@ cheritest_failure_errx(const char *msg, ...)
 	exit(EX_SOFTWARE);
 }
 
+
```
Comment (jrtc27) [inline bin/cheritest/cheritest_util.c:95]:
```suggestion
```

Comment (bsdjhb) [reply bin/cheritest/cheritest_util.c]: Oops, fixed.

**Review by jrtc27 — APPROVED** [review]

**Comment by jrtc27** [comment]
> ld.lld: error: /local/scratch/jenkins/workspace/CheriBSD-pipeline_PR-691/cheribsd-riscv64-hybrid-build/local/scratch/jenkins/workspace/CheriBSD-pipeline_PR-691/cheribsd/riscv.riscv64/tmp/lib/libc.so.7: section header table goes past the end of the file: e_shoff = 0x19cfd0
hmmm @arichardson you saw that recently?

**Comment by arichardson** [comment]
Yeah but it always went away after a rebuild so I haven't debugged it yet.

**Pull request merged**